### PR TITLE
feat: build with rollup

### PR DESCRIPTION
**Pull request by HPrivakos**

### CODE REVIEW

File: static/api-extractor.json [hunk 1]
```diff
@@ -0,0 +1,340 @@
+/**
```
Comment (nicosantangelo) [inline static/api-extractor.json:1]: Should this file be commited? seems like it'll be auto-generated each build right?

File: src/schemas.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export { ChainId } from "@dcl/schemas"
```
Comment (nicosantangelo) [inline src/schemas.ts:1]: this file is breaking the npm run lint command, and subsequently, the build

File: rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+console.log(`production: ${PROD}`)
+
+const plugins = [
+  json({
+    compact: true
+  }),
+  typescript({
+    verbosity: 2,
+    clean: true,    useTsconfigDeclarationDir: true
+
```
Comment (nicosantangelo) [inline rollup.config.js:20]: missing a newline?

File: rollup.config.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  }),
+  resolve({
+    browser: true,
+    preferBuiltins: false
+  }),
+  commonjs({
+    ignoreGlobal: true,
+    include: [/node_modules/],
+    namedExports: {
+      '@dcl/schemas': ['ChainId']
```
Comment (nicosantangelo) [inline rollup.config.js:30]: This is required for each thing we require from dcl/schemas, right? I'm just a bit scared that the config is hidden and someone else updating the lib might break the build inadvertently

**Review by nicosantangelo — CHANGES_REQUESTED** [review]
Let's fix the lint! other than that it looks good

File: static/api-extractor.json (hunk 1 above)
Comment (HPrivakos) [reply static/api-extractor.json:1]: Yes, this should be commited.
That file contains configuration to be used by api-extractor to bundle the types into a single file

File: rollup.config.js (hunk 4 above)
Comment (HPrivakos) [reply rollup.config.js:30]: This is only required for libraries we want to use with the SDK. It bundles @dcl/schemas with the library so it's not required as a dependency anymore (so we don't need to also build @dcl/schemas with the same method to work in the SDK)

**Pull request merged**